test/adapter.js

```diff
@@ -32,7 +29,12 @@ function testFixture(description, filename, subDir = true, generateSourceMap = f
       // Invoke with the env var GENERATE set to regenerate the fixtures.
       if (process.env.GENERATE) {
         expected = output.toRefract();
-        fs.writeFileSync(path.join(__dirname, expectedName), JSON.stringify(expected, null, 2), 'utf8');
+
+        if (generateSourceMap) {
```

Comment:
@danielgtaylor That was difficult to do in swagger-zoo. I think we should leave it here.